source/dub/commandline.d

```diff
@@ -1236,7 +1241,29 @@ class RemoveCommand : FetchRemoveCommand {
 		auto package_id = free_args[0];
 		auto location = dub.defaultPlacementLocation;
 
-		dub.remove(package_id, m_version, location, m_forceRemove);
+		size_t resolveVersion(in Package[] packages) {
+			writeln("Select version of '", package_id, "' to remove from location '", location, "':");
+			foreach(i, pack; packages)
+				writeln(i, ". ", pack.version_);
+			writeln(packages.length, ". ", "all versions");
+			while (true) {
+				writef("> ");
+				auto inp = readln();
+				if (!inp.length) // Ctrl+D
+					return size_t.max;
+				if (inp.length > 1) {
```

Comment:
On Windows it also uses a single `\n`, but it's good I double checked now, because that assumption is also used for the interactive init code (a Windows CI test instance would of course be even better than double-checking, need to look into AppVeyor).

Another possibility to express this in a clear way:
```
inp = inp.stripRight('\n');
if (inp.length == 0) continue;
```